packages/_vue3-migration-test/src/components/scroll/test-scroll.vue

```diff
@@ -0,0 +1,23 @@
+<script setup lang="ts">
+  import MainScroll from '../../../../x-components/src/x-modules/scroll/components/main-scroll.vue';
+  // eslint-disable-next-line max-len
+  import MainScrollItem from '../../../../x-components/src/x-modules/scroll/components/main-scroll-item.vue';
+  const items = Array.from({ length: 24 }, (_, index) => ({ id: `item-${index}` }));
+</script>
+
+<template>
+  <MainScroll>
+    <ul id="rootScroll" class="list" data-test="scroll">
```

Comment:
`id="rootScroll"` is not necessary, right?

Reply:
Not anymore, not really.